internal/pkg/services/user.go

```diff
@@ -103,3 +108,20 @@ func (u *UserService) UpdateUserPassword(email string, password string) error {
 
 	return nil
 }
+
+func (u *UserService) GetAllUsers() (*[]models.User, error) {
+	var users []models.User
+	rows, err := u.DB.Queryx(getAllUsers)
+
+	if err != nil {
+		return nil, err
+	}
+
+	for rows.Next() {
+		var user models.User
+		_ = rows.StructScan(&user)
```

Comment:
You need to always handle errors, you can't just assume that the function will not return an error